pcsx2/GameDatabase.h

```diff
@@ -56,15 +54,10 @@ class GameDatabaseSchema
 		Full
 	};
 
-	struct Patch
-	{
-		std::string author;
-		std::vector<std::string> patchLines;
-	};
+	using Patch = std::vector<std::string>;
```

Comment:
looks like we aren't bubbling up the author metadata for patches into the emulator anymore?  It doesn't really matter to me -- not sure if there are differing opinions on this though.

Reply:
It'll still be in the file? But I don't see much use in loading it at runtime.